satpy/utils.py

```diff
@@ -476,6 +478,19 @@ def _check_yaml_configs(configs, key):
                         pass
     return diagnostic
 
+def _check_package_version(package_name: str) -> Optional[str]:
+    """Check the version of `package_name`.
+
+    Args:
+        package_name (str): the distribution package name.
+
+    Returns:
+        the version number if available else `None`.
+    """
+    try:
+        return importlib.metadata.version(package_name)
+    except importlib.metadata.PackageNotFoundError:
+        return None
 
 def _check_import(module_names):
```

Comment:
It's hard to tell from github's diff display, but is this function used anymore?

Reply:
good point! I've removed the now unused `_check_imports`.